### PR TITLE
Add images workspace consumption based on plan

### LINKED ISSUE #533: Workspace dashboard to follow plan consumption
## Tasks

- [x] Add the workspace corresponding tag next to its name
- [x] Create an `imageAggregate` for a given workspace
- [x] Set progressBar to orange (see mockup) when consumption is above 80%
- [x] Set progressBar to red (see mockup) when consumption is above 100%
- [ ] Implement the workspace limits according to the plan

## Mockup ([here](https://www.figma.com/file/4J9xyfF5v4B98MuKVO0Y8V/LabelFlow-mockups?node-id=4223%3A14405))
image

Depends on #531

### PR DESCRIPTION
**Pull request by Jordanlelay**
## Work performed

- Count number of images per workspace and display a progress bar in settings

## Resolved issues

Closes #533

### CODE REVIEW

File: typescript/web/src/pages/[workspaceSlug]/settings.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ import {
 } from "../../graphql-types/GetWorkspaceDetailsQuery";
 import { useWorkspace } from "../../hooks";
 
-const GET_WORKSPACE_DETAILS_QUERY = gql`
+export const GET_WORKSPACE_DETAILS_QUERY = gql`
```
[the remark this answers is not in the document]
Comment (Jordanlelay) [reply typescript/web/src/pages/[workspaceSlug]/settings.tsx]: Remaining code from earlier, nice catch 👍

**Review by bbenoist — CHANGES_REQUESTED** [review]
@Jordanlelay I've readjusted a little bit the code of the Billing component. Nothing critical but you can take a look at the commit description for details.

- [ ] Other than that, there remains 1 issue to be resolved: since you've removed the Enterprise WorkspacePlan, it's needed to create a DB schema migration and apply it on our dev DB just before we merge this PR in main.

**Comment by bbenoist** [comment]
@Jordanlelay As suggested during the last DSM, and because this PR is OK except for the migration part, I propose you to:
1. Rebase this PR on the latest revision of `main` and fix the conflicts
2. Squash the 3 commits of this PR into a single commit (_with `(#925)` at the end of the title_) and force-push the `feature/plan-consuption` branch.
3. Squash the commits of `feature/trial-period` (#913) and force-push the commit.
4. If the CI goes green on both PR, merge `feature/plan-consumption` into `feature/trial-period` (#913)
5. Push the changes of `feature/trial-period` and make sure that the CI is green.
6. Mob the DB migration part (CC @valerian ) and apply it on the preview DB
7. Verify that everything is OK and merge on main

**Comment by Jordanlelay** [comment]
Closed, will be integrated in #913

**Pull request closed without merging**